PRINCIPLES.md

```diff
@@ -10,22 +10,22 @@ These principles were derived from modern software operations but are rooted in
 
 ## Principles
 
-1. **The principle of declarative desired state**
+1. **The system's desired state is described declaratively**
 
     A system managed by GitOps must have its _Desired State_ expressed declaratively as data in a format writable and readable by both humans and machines.
 
-2. **The principle of immutable desired state versions**
+2. **State declarations are stored as immutable versions**
```

Comment:
```suggestion
2. **Declarations are Version Controlled**
```

Reply:
```suggestion
2. **Declarations are Versioned**
```

Reply:
🤏

Reply:
@lloydchang Thanks!

Re https://github.com/open-gitops/documents/pull/14#pullrequestreview-708672508, I do personally agree the shorter and more memorable/catchy we can make these principle headers the better 🤏 

I wonder though if "versioned" here is enough, whereas "version controlled" seems to me to check all the boxes of this principle. But do you not feel these have different meanings?

> Desired State is stored in a way that supports _versioning_, _immutability of versions_, and _retains a complete version history_.

Reply:
@jlbutler 

```suggestion
2. **Declarations are versioned and Immutable**
```

Reply:
```suggestion
2. **Declarations are audit-proof versioned**
```

Reply:
@jlbutler Updated the PR. 

@heubeck thanks for this suggestion! I haven't added this to the branch yet because I think we'd need more discussion on that first. but I added you as co-author to this commit as well since you're collaborating with us here on GitHub. I'd like us to get this PR merged soon, but we can continue iterating. Are you free to join the next principles meeting on Wednesday?

Reply:
@scottrigby Thank you, I'll try to participate.

Reply:
Regarding my `audit-proof` suggestion: Technically it means the same as the phrases with `immutable` and `versions`, but the statement is much stronger.
Industries with deployment revision logging ("audit") demands may appreciate a less technical terminology.

Reply:
@heubeck this is really interesting. Given how close this PR is to being merged for `v0.2.0`, and how important it is to simplify the headers ASAP, let's discuss this idea for a next iteration 🙏 

Would you be willing to move this idea to a [discussion in the OpenGitOps project repo](https://github.com/open-gitops/project/discussions)? That way there is a place for async discussion on this before and after the next meeting (tomorrow). The discussion could recommend this change from the current PR, and see how that sits with the principles committee and community?
```diff
- 2. **Declarations Are Versioned and Immutable**
+ 2. **Declarations Are Versioned and Audit-Proof**
```

Reply:
Happy to discuss: https://github.com/open-gitops/project/discussions/15